src/components/utils/src/file_system.cc

```diff
@@ -100,16 +100,16 @@ size_t file_system::DirectorySize(const std::string& path) {
 }
 
 // NOTE that boost makes 0777 permissions by default
-std::string file_system::CreateDirectory(const std::string& name) {
+bool file_system::CreateDirectory(const std::string& path) {
   error_code ec;
-  bool success = fs::create_directory(name, ec);
+  bool success = fs::create_directory(path, ec);
```

Comment:
@SKobziev const?

Reply:
Yes. If I understand your question rightly, the parameter "path" is const because other methods wait for const data for the first parameter. It's a good way of saving "path" as unchanged. Otherwise could you please clarify your question?

Reply:
@SKobziev `const bool success`...

Reply:
@AByzhynar It was done